library/java/io/envoyproxy/envoymobile/Envoy.java

```diff
@@ -45,7 +45,7 @@ public static void load(Context context) {
   // initialization with the configuration provided. If the Envoy native library and its
   // dependencies haven't been loaded and initialized yet, this will happen lazily when
   // the first instance is created.
-  public Envoy(Context context, String config) {
+  public Envoy(final Context context, final String config) {
```

Comment:
nice

Reply:
This is probably because these files are going through to the kotlin bazel rules. Wont be a problem for long!

Reply:
I mean I think it's a good thing